packages/flutter/test/widgets/semantics_test.dart

```diff
@@ -706,7 +706,7 @@ void main() {
       ],
     );
     expect(semantics, hasSemantics(expectedSemantics, ignoreId: true));
-  });
+  }, skip: true); // [intended] https://github.com/flutter/flutter/issues/127617
```

Comment:
I vaguely remember there is another test in custom painter you need to disable. Also, I don't think you should put a [intended] here. Unless I have outdated memory, the intended should only be used when it is permanent. For example, a test that only target web platform or etc

Reply:
Ah I see! Will update the comment and skip the test in custom_painter.dart. Thanks a lot!